packages/ui/src/composables/useElementBackground.stories.ts

```diff
@@ -0,0 +1,198 @@
+import { defineComponent, computed, ref } from 'vue'
+import { useElementBackground } from './useElementBackground'
+import { useCurrentElement } from './useCurrentElement'
+import { VaButton } from '../components/va-button'
+import UseElementBackgroundDummy from './UseElementBackgroundDummy.vue'
+import { within } from '@storybook/testing-library'
+import { sleep } from '../utils/sleep'
+import { expect } from '@storybook/jest'
+
+export default {
+  title: 'composables/useElementBackground',
+}
+
+export const Default = () => ({
+  components: { VaButton, UseElementBackgroundDummy },
+  data () {
+    return {
+      background: '#000',
+      color: null,
+    }
+  },
+  template: `
+  <UseElementBackgroundDummy
+    :style="{ background }"
+    @update:color="color = $event"
+  />
+  <p>Color detected: <span data-testid="color">{{color}}</span></p>
```

Comment:
So far we're having clear blocks for stories.

That is:
[ins]
[story target]
[outs]
[controls]

I suggest we stick to that structure.
I checked what storybook has to offer (i.e. if we can use their controls) - and I found nothing useful. If it's doable - it will be twice as complex and less flexible.

Reply:
Can you change it later? I didn't get it :_)